Assembler/assembler.c

```diff
@@ -139,7 +139,10 @@ bool classifiedCMD( char *line, bool *isLableFound, bool *isNoteFound, bool *isD
 }
 
 bool isLineValid( char *line ) {
-    return TRUE; /* TODO change */
+    if ( line[0] == '\n' || line[0] == '#' ) {
```

Comment:
FYI you can return:
`return line[0] != '\n' && line[0] != '#'`
instead the if,